### ISSUE TITLE
Remove dependency on ggalt

### ISSUE THREAD

**Issue by MilesMcBain** [njtierney/ggmissing#19]
ggalt needs to build with PROJ.4 which means ggmissing will fail when installing on a Linux system if PROJ.4 is not installed. This happened to me today. A quick google suggests this library has something to do with map projections... so nothing missing data related. This is also why Travis build is failing.

I recreated geom_lollipop for `gg_missing_var` using stock ggplot2. I started on `gg_missing_case` but realised I have no idea what that plot is supposed to look like, and since I won't install ggalt, I can't find out. 

See: https://github.com/njtierney/ggmissing/blob/miles/R/gg_missing_var.R

**Comment by njtierney** [comment]
Looks good!

Could you submit a pull request, and add a bullet to NEWS.md?

**Comment by njtierney** [comment]
Oops, sorry.

`gg_missing_case` looks like this:

```r
library(ggplot2)
library(ggmissing)
ggplot(data = summary_missing_case(airquality),
       aes(x = case,
           y = n_missing)) +
  geom_bar(stat = "identity") +
  geom_point() +
  # ggalt::geom_lollipop(horizontal = TRUE) +
  # geom_line() + 
  labs(x = "Case",
       y = "# Missing") +
  theme_minimal() + 
  coord_flip() 
```

You can reorder it, as in `gg_missing_var`, but it looks like ggplot is treating these as factors, or similar.

```r

ggplot(data = summary_missing_case(airquality),
       aes(x = reorder(case, n_missing),
           y = n_missing)) +
  geom_bar(stat = "identity") +
  geom_point() +
  # ggalt::geom_lollipop(horizontal = TRUE) +
  # geom_line() + 
  labs(x = "Case",
       y = "# Missing") +
  theme_minimal() + 
  coord_flip() 

}
```

**Comment by MilesMcBain** [comment]
Resolved by #21
